misc/scripts/install-local.sh

```diff
@@ -961,9 +961,13 @@ mkdir -p "${SRCDIR}"
 if [[ -n $patch ]]; then
     fancy_message info "Downloading patches"
     mkdir -p PACSTALL_patchesdir
+    # NOTE: not using --output-dir,
+    # since Buster/Focal include a version of curl w/o that option.
+    pushd PACSTALL_patchesdir > /dev/null || return 1
```

Comment:
Let's add an error message on that `return 1` just so people know what went wrong if it does go wrong.

Reply:
You can commit to my branch. I'm bad at messages

Reply:
It's your PR, just write down what the error actually is.